src/aoemotebutton.cpp

```diff
@@ -25,10 +24,45 @@ void AOEmoteButton::set_image(QString p_image, QString p_emote_comment)
         "QToolTip { color: #000000; background-color: #ffffff; border: 0px; }");
   }
   else {
-    this->setText(p_emote_comment);
-    this->setStyleSheet("QPushButton { border-image: url(); }"
-                        "QToolTip { background-image: url(); color: #000000; "
-                        "background-color: #ffffff; border: 0px; }");
+    if (p_image.contains("_off") || p_image.contains("_on")) {
```

Comment:
Wait a minute... doesn't this mean that if either button is not present, then the missing button will always be darker? This generator should only work to generate `_on` images, which, by convention, are darker than the `_off` ones.

Reply:
Done